### ISSUE TITLE
Resin.io compatible deployment

### ISSUE THREAD

**Issue by SloMusti** [ch2i/LoraGW-Setup#1]
It would be great if this could be used with resin.io much like many other configurations: https://github.com/kersing/ttn-resin-gateway-rpi

**Comment by hallard** [comment]
Yeah, for sure I would love that, but I'm not an expert of resin.io and my schedule is really tight. So if any resin.io expert want to help, he's welcome.
